package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "weather-app-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "jest",
+    "start": "node dist/server.js",
+    "build": "tsc"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
what is ISC?

Reply:
It was automatically created when I initialized the project with git init -y. Many open-source projects, especially those in the Node.js and JavaScript ecosystem, use the ISC License due to its simplicity and compatibility with other licenses, being an open-source software license that allows for the free use, modification, and distribution of the software under certain conditions. Users are granted the freedom to use, modify, and distribute the software with the requirement to include the original copyright notice in any distributions.